extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -1,3 +1,19 @@
+/**
+ * Randomize array element order in-place.
+ * Using Durstenfeld shuffle algorithm.
+ *
+ * @see https://stackoverflow.com/a/12646864
+ * @see https://en.wikipedia.org/wiki/Fisher-Yates_shuffle#The_modern_algorithm
+ */
+function shuffleArray(array) {
```

Comment:
Is there not a jQuery method to do this?

Reply:
It's underscore/lodash